pcdsdevices/beam_stats.py

```diff
@@ -7,10 +9,17 @@
 from .signal import AvgSignal
 
 
+logger = logging.getLogger(__name__)
+
+
 class BeamStats(BaseInterface, Device):
+    # Pulse energy (in mJ)
     mj = Cpt(EpicsSignalRO, 'GDET:FEE1:241:ENRC', kind='hinted')
```

Comment:
Please add `doc=` items, as they'll render nicely in the documentation:

```suggestion
    mj = Cpt(EpicsSignalRO, 'GDET:FEE1:241:ENRC', kind='hinted', doc='Pulse energy (in mJ)')
```

Reply:
Added some in e13f86b :)